src/main/java/GameSetup.java

```diff
@@ -49,29 +49,22 @@ private void setPlayerTwo(String choice) {
     }
 
     private void findFirstPlayer() {
-        write("Who goes first?  Please choose X or O");
+        display.promptForFirstGo();
         String choice = loopForValidSymbol();
         Board board = new Board();
-        if (choice.equalsIgnoreCase("O") && playerTwo.getSymbol().equals("O")) {
+        if (choice.equalsIgnoreCase("O") && playerTwo.getMark().equals("O")) {
             game = new Game(playerTwo, playerOne, board);
-        } else if (choice.equalsIgnoreCase("X") && playerTwo.getSymbol().equals("X")) {
+        } else if (choice.equalsIgnoreCase("X") && playerTwo.getMark().equals("X")) {
```

Comment:
Maybe playerOne and playerTwo would be better named as playerX and playerO, because sometimes platerOne is the second player which could be confusing.